src/styles/tickets/frontend.pcss

```diff
@@ -3,308 +3,339 @@
  *
  * @since  0.3.0-alpha
  */
-.tribe-block__tickets {
+.tribe-common.tribe-block__tickets {
+	border: 1px solid var(--color-border-secondary);
+	border-radius: 4px;
 	margin-top: 30px;
 	margin-bottom: 30px;
+	padding: 1rem;
 }
 
-.tribe-block__tickets__item {
-	display: flex;
-	flex-wrap: wrap;
-	background: #fff;
-	border: 1px solid #dfdfdf;
-	border-bottom-width: 0;
+ .tribe-common {
 
-	&:last-of-type {
-		border-bottom-width: 1px;
-	}
-}
+	 .tribe-block__tickets__title {
+		 font-weight: var(--font-weight-regular);
+	 }
 
-.tribe-block__tickets__item__icon {
-	background: #f9f9f9;
-	flex: 1 1 70px;
-	padding: 25px 10px;
-	text-align: center;
+	.tribe-block__tickets__item {
+		display: flex;
+		flex-flow: row wrap;
+		background: var(--color-background);
+		border-bottom: 1px solid var(--color-border-secondary);
+		border-top: 1px solid var(--color-border-secondary);
+		padding: .875em 0;
```

Comment:
em. also let's use spacer vars if possible.

Reply:
Yeah, these are going away - ignore this file.